tests/testthat/test-cube-with-array-selected-array.R

```diff
@@ -0,0 +1,158 @@
+context("Cubes with categorical array and multiple response (deprecated selected_array)")
```

Comment:
Why not move all of the old-way tests below into this file?

Reply:
Thanks, I missed removing a few old-way tests in `test-cube-with-array.R` when I made this file. Those have been removed (and checked to confirm they were in the old-way file) so `test-cube-with-array.R` is all `as_selected` and `test-cube-with-array-selected-array.R` is all `selected_array`